File: python/numero.py
numero = int(input("ingrese un numero: "))

if numero > 1:
    for i in range(2, int(numero ** 0.5) + 1):
        if numero % i == 0:
            print("no es un numero primo")
            break
        else:
            print("es un numero primo")
else:
    print("no es primo")

File: javascript/numero.js
let numero = parseInt(prompt("Ingrese un numero: "));
let esPrimo = true;
if(numero <= 1){
    esPrimo = false;
}
else{
    for(let i = 2; i <= Math.sqrt(numero); i++){
        if(numero % i === 0){
           esPrimo =  false;
           break;
        }
    }
    
}

if(esPrimo){
    console.log("es un numero primo")
}
else{
    console.log("no es un numero primo")
}